src/components/App.tsx

```diff
@@ -3670,7 +3690,9 @@ class App extends React.Component<AppProps, AppState> {
         if (
           (hitElement === null || !someHitElementIsSelected) &&
           !event.shiftKey &&
-          !pointerDownState.hit.hasHitCommonBoundingBoxOfSelectedElements
+          !pointerDownState.hit.hasHitCommonBoundingBoxOfSelectedElements &&
+          // checking isMidPoint since "hitTestLinear" returns false when adding midpoint outside editor
```

Comment:
This works fine but will check for a better alternative for this

Reply:
can't repro what it solves — what are the steps?

Reply:
Actually you are right, it's working fine now. Earlier it wasn't, I wasn't able to drag the phantom point due to hit box failing. Will remove this condition as its working as expected. Will check how this got fixed